### ISSUE TITLE
Thanks for trying our work

### ISSUE THREAD

**Issue by ironjr** [lucidrains/grokfast-pytorch#2]
Wow, I just found this nice repository. As an author of Grokfast, I am very happy to have such an attention from the beloved ML community.

If there is any issue regarding the theory, the implementation, and the experiments of Grokfast, please feel free to ask me anything _here_, through [my twitter](https://twitter.com/_ironjr_), or in [our official repository](https://github.com/ironjr/grokfast).

Many thanks!

Jaerin

**Comment by lucidrains** [comment]
@ironjr indeed, thanks for the interesting paper in this somewhat unexplored area!

**Comment by ironjr** [comment]
Thank you for your kind response!
